azure-iot-hub/azure/iot/hub/iothub_registry_manager.py

```diff
@@ -74,7 +74,17 @@ def __del__(self):
         """
         self.amqp_svc_client.disconnect_sync()
 
-    def create_device_with_sas(self, device_id, primary_key, secondary_key, status, iot_edge=False):
+    def create_device_with_sas(
+        self,
+        device_id,
+        primary_key,
+        secondary_key,
+        status,
+        iot_edge=False,
+        status_reason=None,
```

Comment:
> status_reason=None [](start = 7, length = 19)

You need to update the docstrings in this file for all the parameters you added.  The docstrings get run through a tool and published for end users, so these need extra attention in terms of content and also in terms of format.  I assume it will be lots of copy/paste, but I'm not sure how you deal with parent_scopes having 2 different types.  I'm sure there's syntax for this; hopefully you can find a good example.

Reply:
Updated the docstrings. @cartertinney , can you confirm the formatting is okay?